test_scripts/API/VehicleData/StabilityControlsStatus/005_GetVehicleData_StabilityControlsStatus_EscSystemOnly.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data
+-- (with escSystem param only) via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+--
+-- Steps:
+-- 1) App send GetVehicleData (with stabilityControlsStatus = true) request to SDL
+-- SDL does:
+--  - send VehicleInfo.GetVehicleData (with stabilityControlsStatus = true) request to HMI
+-- HMI sends VehicleInfo.GetVehicleData response "SUCCESS"
+--   with next data only (escSystem = "ON")
+-- SDL does:
+--  - send GetVehicleData response with (success: true resultCode: "SUCCESS") 
```

Comment:
@ydementieiev Rephrease it please to : 
`SDL sends GetVehicleData response with data received from HMI and (success: true resultCode: "SUCCESS") to the app`

Reply:
@AByzhynar thank you , [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/14/commits/1083dd2dbfb083e5b89ac7e7cf8bd751979f060a)